templates/CRM/Core/BillingBlock.tpl

```
@@ -79,6 +79,16 @@
     </fieldset>
   {/if}
 </div>
+
+{literal}
+  <script type="text/javascript">
```

This will fail silently if those fields are not on the page won't it?